src/routes/people/+page.ts

```diff
@@ -0,0 +1,7 @@
+import getTeam from '$lib/people/team/team';
+
+export async function load() {
+    return {
+        team: await getTeam(),
+    };
```

Comment:
This is a personal nitpick on code style; feel free to ignore if you disagree.
```suggestion
    return { team: await getTeam() };
```